### PR TITLE
NumberTag & StructObjectListEdit.js

### PR DESCRIPTION
**Pull request by msladek**
https://synjira.atlassian.net/browse/CELDEV-866
https://synjira.atlassian.net/browse/CELDEV-867

### CODE REVIEW

File: component/src/main/java/com/celements/structEditor/fields/NumberTagPageType.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    return VIEW_TEMPLATE_NAME;
+  }
+
+  @Override
+  public Optional<String> tagName() {
+    return Optional.of("input");
+  }
+
+  @Override
+  public void collectAttributes(AttributeBuilder attrBuilder, DocumentReference cellDocRef) {
```
Comment (fpichler) [inline component/src/main/java/com/celements/structEditor/fields/NumberTagPageType.java:35]: unit tests missing

File: web-module/src/main/webapp/resources/structEditJS/StructObjectListEdit.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+      if (objNb >= 0) {
+        // ^ in front of the object number is the delete marker
+        nameParts[1] = "^" + nameParts[1];
+        objElem.setAttribute("name", nameParts.join('_'));
+      }
+    }    
+    return objNb;
+  };
+
+  $j(document).ready(observeCreateObject);
```
Comment (fpichler) [inline web-module/src/main/webapp/resources/structEditJS/StructObjectListEdit.js:71]: struct editor forms may be loaded async. Thus please register additionally on 'celements:contentChanged'.

File: web-module/src/main/webapp/resources/structEditJS/StructObjectListEdit.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
+(function(window, undefined) {
```
Comment (fpichler) [inline web-module/src/main/webapp/resources/structEditJS/StructObjectListEdit.js:1]: please refactor to a js-class

**Review by fpichler — CHANGES_REQUESTED** [review]

File: web-module/src/main/webapp/resources/structEditJS/StructObjectListEdit.js (hunk 3 above)
Comment (msladek) [reply web-module/src/main/webapp/resources/structEditJS/StructObjectListEdit.js:1]: CELDEV-865

**Review by fpichler — APPROVED** [review]

**Pull request merged**